torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -811,6 +812,7 @@ ProcessGroupNCCL::ProcessGroupNCCL(
             << waitTimeoutDumpInMilSec_
             << ", TORCH_NCCL_DESYNC_DEBUG: " << desyncDebug_
             << ", TORCH_NCCL_ENABLE_TIMING: " << enableTiming_.load()
+            << ", TORCH_NCCL_COMPUTE_DURATION: " << computeDuratoin_
```

Comment:
i dont see any logic to error if someone sets compute_duration=True and timing_enabled=False.  That should be an error, right?  (or else it should be documented that setting 'compute_duration' automatically enables timing.

Reply:
Yes, decided to go the later option to make user's life easier such that they just need t specify one env to enable compute_duration